vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationMain.java

```diff
@@ -75,7 +74,7 @@ public ExperimentInformationMain(
     this.projectInformationService = projectInformationService;
     this.experimentInformationService = experimentInformationService;
     layoutComponent();
-
+    addListeners();
```

Comment:
Why move this to the constructor?

Reply:
The previous behaviour was to add the listeners everytime a beforeEnterEvent happens, which is unnecessary if the listeners can be instantianted once in the constructor with the same behaviour.